### PR TITLE
Conditionally instantiate File Managers on packages

### PR DESCRIPTION
**Pull request by godfreyyeung**
Only create File Managers on Applicant Packages,
not on Planning Packages

### CODE REVIEW

File: client/app/models/package.js [hunk 1]
```diff
@@ -2,16 +2,36 @@ import Model, { attr, belongsTo } from '@ember-data/model';
...
+      const fileQueue = this.fileQueue.create(this.id);
+
+      this.fileManager = new FileManager(
+        this.id,
+        this.documents,
+        [],
+        fileQueue,
+      );
+    }
   }
```
Comment (allthesignals) [inline client/app/models/package.js:35]: Did you see the deprecation warning link I sent? If we're going to refactor this, we should just refactor it away from the deprecated API.

Comment (godfreyyeung) [reply client/app/models/package.js:35]: Yeah, I wanted to do this in small chunks tho! I agree. 
I looked into using the afterModel hook, but felt it was too many changes in one commit

Comment (allthesignals) [reply client/app/models/package.js:35]: I'm not sure why this change is necessary. We don't need to optimize for this. I remember we talked about this, it's a lot of code to add just to add an optimization that we're not sure is even an issue to begin with.

Comment (godfreyyeung) [reply client/app/models/package.js:35]: Hmm ok, I can close this.

**Pull request closed without merging**